Content.Shared/Actions/SharedActionsSystem.cs

```diff
@@ -163,6 +165,14 @@ public void StartUseDelay(EntityUid? actionId)
         Dirty(actionId.Value, action);
     }
 
+    public void OnRejuventate(EntityUid uid, ActionsComponent component, RejuvenateEvent args)
```

Comment:
Is there a reason for it being public?
```suggestion
    private void OnRejuventate(EntityUid uid, ActionsComponent component, RejuvenateEvent args)
```